tests/test_bios_password.py

```diff
@@ -64,17 +71,24 @@ def test_rm_bios_pass_ok(self, mock_get, mock_post):
         responses = INIT_RESP + responses_get
         self.set_mock_response(mock_get, 200, responses)
         self.set_mock_response(mock_post, 200, JOB_OK_RESP)
+        self.set_mock_response(mock_delete, 200, "OK")
         self.args = [self.option_arg, "--old-password", "new_pass"]
-        _, err = self.badfish_call()
```

Comment:
might get a lint issue with this one
try running `black .` and `flake8 --ignore D203,E501,E203,W503` on  /src and /tests